src/classiceditorui.js

```diff
@@ -29,20 +29,24 @@ export default class ClassicEditorUI extends BoxedEditorUI {
 		super( editor );
 
 		/**
-		 * Toolbar controller.
+		 * Editable UI controller.
 		 *
 		 * @readonly
-		 * @member {ui.toolbar.Toolbar} editor-classic.ClassicEditorUI#toolbar
+		 * @member {ui.editableUI.EditableUI} editor-classic.ClassicEditorUI#editable
 		 */
-		this.toolbar = this._createToolbar();
+		this.editable = this._createEditableUI();
+	}
 
+	init() {
 		/**
-		 * Editable UI controller.
+		 * Toolbar controller.
 		 *
 		 * @readonly
-		 * @member {ui.editableUI.EditableUI} editor-classic.ClassicEditorUI#editable
+		 * @member {ui.toolbar.Toolbar} editor-classic.ClassicEditorUI#toolbar
 		 */
-		this.editable = this._createEditableUI();
+		this.toolbar = this._createToolbar( this.editor.ui.view.element );
```

Comment:
> In case of ClassicEditorUi it's this.editor.ui.view.element, which is unavailable in ClassicEditorUi#constructor() yet.

Why isn't it? I see that `this.editor.ui.view` is set in `ClassicEditor` after setting `this.editor.ui`... this doesn't make sense – why isn't it passed to `ClassicEditorUI` like in all other controllers? In fact – the whole chain of editor ui classes has this wrong.

> Either way, there's some job to be done in ClassicEditorUi#init() and I'm wondering if limiterElement should be part of component's state. This was, in fact, what the first implementation looked like but I backed out of this approach. IMO StickyToolbarView is way too static, i.e. it depends on its _placeholderElement to work properly, which is injected once where the toolbar is created and never moves in DOM, so making limiterElement part of the model makes little sense, because the toolbar would not be able to stick somewhere else anyway, it's bound to the spot it's been initialised, so why bother.

OK, so I can agree with that, but it means that this component will never support realigning to different elements. And this sounds like a realistic use case, because with multiple editables you'll like if the toolbar followed them.

So by putting the limitter in the model we should be more future-proof and for now we can document it as static (only settable once).